src/model/scripts/bpe_training/train.py

```diff
@@ -0,0 +1,77 @@
+from datasets import load_dataset
+from tokenizers import ByteLevelBPETokenizer
+from nlpo3 import segment, load_dict
+import argparse
+import os
+
+SPECIAL_TOKENS = [
+    "<s>",
+    "<pad>",
+    "</s>",
+    "<unk>",
+    "<mask>",
+]
+
+DICT_FILE = f"{os.path.dirname(__file__)}/words_th.txt"
+DICT = "dict"
+
+load_dict(DICT_FILE, DICT)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--output_path", type=str, help="path of tokenizer file", required=True
+    )
+    parser.add_argument(
+        "--vocab_size", type=int, default=50000, help="max vocab of tokenier"
+    )
+    parser.add_argument("--data_path", type=str, help="path of datasets", required=True)
+    parser.add_argument(
+        "--data_type",
+        default=None,
+        help="data type of dataset if None will be Huggingface format",
+    )
+
+    args = parser.parse_args()
+
+    # load dataset
+    if args.data_type is None:
```

Comment:
I think this function is as same as PR #189. Can we refactor or do something after merged this PR?